app/scripts/controllers/channelsmodal.js

```diff
@@ -1199,3 +1198,15 @@ app.controller('channelAlertsCtrl', function ($scope, Api) {
   /****************************************************************/
 
 });
+
+// nested controller for the channel settings tab
+app.controller('channelSettingsCtrl', function ($scope) {
```

Comment:
I think this controller need to be removed, it seems to be a repeat from line 783. I may have been a mistake in the reverting.